### ISSUE TITLE
Let it Snow: "snowflakes: " text should be a label element for the select

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#239]
This is for the snowflake count at the bottom of the page.

**Comment by alrra** [comment]
Closing in favor of #250.
